reference/replay.tracker.events.md

```diff
@@ -135,12 +135,24 @@ Always identical to `m_upkeepPlayerId`?
 
 ## 6. m_unitTagIndex (number)
 
-Starts at 1 with 1 increments?
+m_unitTagIndex and m_unitTagRecycle are used to determine the unique identifier of the unit within the game.
 
-Simple unit counter?
+The formula to determine this value, known as m_tag, is m_unitTagIndex << 18 + m_unitTagRecycle. << is the bitwise shift operator
+
+m_tag is useful to determine what unit was targeted by a skill, for example in this NNet.Game.SCmdEvent present in the replay.game.events file:
+```
+{"_eventid": 27, "m_unitGroup": null, "_event": "NNet.Game.SCmdEvent", "m_abil": {"m_abilLink": 147, "m_abilCmdIndex": 0, "m_abilCmdData": null}, "_gameloop": 2365, "_bits": 264, "m_data": {"TargetUnit": {"m_snapshotControlPlayerId": 6, "m_snapshotPoint": {"y": 235853, "x": 579289, "z": 32441}, "m_snapshotUpkeepPlayerId": 6, "m_timer": 0, "m_targetUnitFlags": 111, "m_snapshotUnitLink": 281, "m_tag": 45350913}}, "_userid": {"m_userId": 5}, "m_cmdFlags": 2097408, "m_sequence": 451, "m_otherUnit": null}
+```
+we see that the unit with tag 45350913 was the target of the ability id 147 casted by player 5.
+
+It is also possible retrieve m_unitTagIndex and m_unitTagRecycle from m_tag, by using the following formula:
+m_unitTagIndex = (m_tag >> 18) & 0x00003fff
+m_unitTagRecycle = (m_tag) & 0x0003ffff
```

Comment:
Could you reference the note about the built-in functions at the top of the file instead? Maybe improve that note because it's not that clear?